src/readii/process/images/crop.py

```
@@ -0,0 +1,126 @@
+        msg = f"Label {label} not present in mask. Must be one of {stats.GetLabels()}"
+        logger.exception(msg)
+        raise ValueError(msg)
+
+    # Generate bounding box based on the specified crop method
+    match crop_method:
+        case "bounding_box":
+            # Generate a bounding box around a mask
+            crop_box = RegionBox.from_mask_bbox(mask, label)
+        
+        case "centroid":
+            if resize_dimension == None:
```

Ruff (0.8.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)